components/dropdown/test/dropdown-content.test.js

```diff
@@ -481,27 +448,27 @@ describe('d2l-dropdown', () => {
 		it('vertical offset should update if set without px', async() => {
 			content.setAttribute('vertical-offset', 100);
 			await nextFrame();
-			expect(content._verticalOffset).to.equal(100);
+			expect(content._offset).to.equal(100);
```

Comment:
The `PopoverMixin` internal value for this is `_offset` (to allow for both block and inline positioning options).

Reply:
Sorry I meant the unit tests and not vdiffs (I realize it would be unrealistic to maintain both); something like mocking the flag so that there are unit test versions testing both states